src/AIPlayer.java

```diff
@@ -4,8 +4,14 @@
 import static java.util.Optional.of;
 
 public class AIPlayer implements Player {
+    private final int START_POSITION = 1;
+    private Random random;
     private Throw aThrow;
```

Comment:
We talked about this, but now that I'm looking at it I would probably change the name of the Throw class. Having to name variables like aThrow is annoying enough for me to find a different name.